src/fq.rs

```diff
@@ -186,6 +208,10 @@ const R2: Fq = Fq([
 ]);
 
 impl Fq {
+    pub fn new(limbs: [u64; 4]) -> Fq {
```

Comment:
We don't want a publicly accessible method like this, since it would allow downstream users to violate the API invariant that any Fq element represents a canonical field element in Montgomery form.